### PR TITLE
fix #78722 thgss.com

### LINKED ISSUE #78722: thgss.com
### Issue URL (Anti Adblock Script)
 [http://thgss.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fthgss.com)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | TDI
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Self-destructing third-party cookies (180), <br>Hide your IP address (123.45.67.091)
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Language-specific:</b><br/> Liste AR, <br/>Icelandic ABP List, <br/>AdBlockID, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polish Anti Adblock Filters, <br/>Dandelion Sprout's Nordic Filters, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Dutch
Userscripts: | disabled
Referrer: | [http://thgss.com/AR1/index.php?p=download](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fthgss.com%2FAR1%2Findex.php%3Fp%3Ddownload)

### PR DESCRIPTION
**Pull request by wout-codes**
#78722 
Anti-adblock caused by DNS.

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 ukrainianwall.com##.privacy-policy
+thgss.com###cookiesUse
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:95]: I can't download a file without accepting cookies.
1. Exchange this rule to `thgss.com#%#//scriptlet("set-cookie", "acceptUsingCookies", "yes")`.
2. Add `thgss.com#@##acceptCookies` with task comment to the `cookies_whitelist.txt`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -572,6 +572,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78722
+thgss.com##.blockerAlert
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:576]: Add a comment that this detection is caused by DNS.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Ops, added the exception rule directly https://github.com/AdguardTeam/AdguardFilters/commit/aaffe7194c6a944f6a93ef788ba0e043ae9027a8

**Comment by wout-codes** [comment]
I was adding it, but you merged it.

**Pull request merged**